src/Propel/Common/Config/Loader/IniFileLoader.php

```diff
@@ -58,7 +59,7 @@ public function supports($resource, $type = null)
      */
     public function load($file, $type = null)
     {
-        $ini = parse_ini_file($this->getPath($file), true);
+        $ini = parse_ini_file($this->getPath($file), true, INI_SCANNER_RAW);
```

Comment:
It could have been INI_SCANNER_TYPED to avoid additional logic in parseKey() but it would have broke compatibility with PHP 5.5